client/packages/portal-client/src/framework-config/portal-framework-config.tsx

```diff
@@ -34,6 +36,19 @@ export function createPortalFramework(portalConfig: PortalConfig) {
 
 		addPortalClient(config, portalConfig.portalClient.client);
 
+		/** Enabling application insight module */
+		if (portalConfig.applicationInsight) {
+			enableTelemetry(config, {
+				connectionString: portalConfig.applicationInsight.connectionString,
```

Comment:
Rename applicationInsight to applicationInsights

Reply:
will do ;)